colobot-base/src/common/CMakeLists.txt

```diff
@@ -43,6 +43,8 @@ target_sources(Colobot-Base PRIVATE
     resources/sdl_memory_wrapper.h
     resources/sndfile_wrapper.cpp
     resources/sndfile_wrapper.h
+    resources/physfs_utils.cpp
+    resources/physfs_utils.h
```

Comment:
I think it's supposed to be in alphabetical order.

Reply:
Thanks. Done.